config/configuration.json

```diff
@@ -223,6 +223,7 @@
     }
   },
   "OneMemberRelationCheck": {
+    "relations.skip": ["type->person,multipolygon"],
```

Comment:
Is this any different than `tags.filter` ?

Reply:
Nope! 🤦‍♂️ good catch

Reply:
Tests are still worth adding, you'll just need to pass in a custom config